internal/pkg/util/loop/loop.go

```diff
@@ -29,14 +29,15 @@ func (loop *Device) AttachFromFile(image *os.File, mode int, number *int) error
 		if fi, err := os.Stat(path); err != nil {
 			dev := int((7 << 8) | device)
 			esys := syscall.Mknod(path, syscall.S_IFBLK|0600, dev)
```

Comment:
To match other block devices, this should be:  `syscall.S_IFBLK|0660`   Ownership:  `root:disk`